addons/website/static/src/snippets/s_searchbar/000.xml

```diff
@@ -1,54 +1,71 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
 
-<!-- Search Bar autocomplete item -->
+<t t-name="website.search_image_item">
+    <div class="d-flex align-items-center justify-content-baseline flex-column">
+        <img t-if="result['image_url']" t-att-src="result['image_url']"
+            class="flex-shrink-0 o_image_64_contain"/>
+        <i t-else="" t-attf-class="o_image_64_contain text-center pt16 fa #{result['_fa']}"/>
+    </div>
+</t>
+
+<t t-name="website.search_items_page">
+    <div class="row g-0 p-2">
+        <t t-foreach="bucket.data" t-as="result" t-key="line_index">
+            <div class="col-12 col-md-6 col-lg-4">
+                <a t-att-href="result['website_url']"
+                    class="o_search_result_item p-2 text-wrap w-100 h-100
+                    d-flex align-items-center justify-content-baseline
+                    flex-nowrap text-truncate dropdown-item">
```

Comment:
Do we really need all this classes?
<img width="1204" height="86" alt="image" src="https://github.com/user-attachments/assets/e2a2f590-3803-466b-a525-d5f4470e735f" />
I guess we can remove ```text-wrap flex-nowrap dropdown-item``` and ```justify-content-baseline```(as I said earlier), maybe ```text-truncate``` also